ui/ozone/platform/wayland/test/mock_buffer.cc

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "ui/ozone/platform/wayland/test/mock_buffer.h"
+
+namespace wl {
+
+const struct wl_buffer_interface mock_wl_buffer_impl = {&DestroyResource};
+
+MockBuffer::MockBuffer(wl_resource* resource, std::vector<base::ScopedFD>&& fds)
```

Comment:
rvalue doesn't seem to be needed here, unless it insists to use move semantics.

Reply:
It does, that was the purpose.